keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ApplicationController.kt

```diff
@@ -64,4 +67,18 @@ class ApplicationController(
       "hasManagedResources" to resourceRepository.hasManagedResources(application),
       "hasDeliveryConfig" to hasDeliveryConfig)
   }
+
```

Comment:
@erikmunson these methods on the application controller are for pausing/resuming an app